### PR TITLE
FAB-17214 Update azure yaml files

### PR DESCRIPTION
**Pull request by rameshthoomu**
Update Azure templates to always CopyLogs and artifacts, whether tests fail or succeed.

Signed-off-by: rameshthoomu <rameshbabu.thoomu@gmail.com>

### CODE REVIEW

**Review by scottz64 — COMMENTED** [review]
why not use "always()" for all of them? what is the difference?

**Comment by rameshthoomu** [comment]
> why not use "always()" for all of them? what is the difference?

This step will always run, even if the pipeline is canceled. In our case, it is not required. Will update the script.

**Comment by rameshthoomu** [comment]
@Vijaypunugubati @suryalnvs @bharadwajambati95

File: ci/azure-pipelines-pte-daily.yml [hunk 1]
```diff
@@ -39,12 +39,15 @@ jobs:
       mkdir -p logs
       ls logs
       cp tools/PTE/CITest/Logs/*.* logs
+    condition: always()
     displayName: Copy Logs
   - task: PublishBuildArtifacts@1
+    condition: or(succeeded(), failed())   # publish either way
     inputs:
       pathtoPublish: 'logs'
```
Comment (suryalnvs) [inline ci/azure-pipelines-pte-daily.yml]: Copy Logs task can be removed and PublishBuildArtifacts@1 can be used to publish Logs directly without copying like pathtoPublish: 'tools/PTE/CITest/Logs'

**Review by scottz64 — APPROVED** [review]
LGTM. Once @suryalnvs is happy with it, we can merge this.

File: ci/install_deps.yml [hunk 2]
```diff
@@ -20,10 +27,5 @@ steps:
       versionSpec: $(NODE_VER)
     displayName: Install Node.js
 
-  - task: UsePythonVersion@0
-    inputs:
-      versionSpec: $(PYTHON_VER)
-    displayName: Install Python
-
   - script: python -m pip install --upgrade pip setuptools pytest
```
Comment (suryalnvs) [inline ci/install_deps.yml]: do we still need this?

**Pull request merged**